hack/test-integration.sh

```diff
@@ -54,24 +51,22 @@ package="${OS_TEST_PACKAGE:-test/integration}"
 tags="${OS_TEST_TAGS:-integration !docker etcd}"
 
 export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
-TMPDIR="${TMPDIR:-"/tmp"}"
-export BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-integration}"
-rm -rf ${BASETMPDIR} | true
-mkdir -p ${BASETMPDIR}
-
+os::util::environment::setup_tmpdir_vars "test/integration/"
+rm -rf ${BASE_TEMPDIR} | true
+mkdir -p ${BASE_TEMPDIR}
+os::util::environment::setup_etcd_vars
 
 echo
 echo "Test ${package} -tags='${tags}' ..."
 echo
 
 # setup the test dirs
-export ETCD_DIR=${BASETMPDIR}/etcd
-etcdlog="${BASETMPDIR}/etcd.log"
+etcdlog="${ETCD_DATA_DIR}/etcd.log"
 testdir="${OS_ROOT}/_output/testbin/${package}"
 name="$(basename ${testdir})"
 testexec="${testdir}/${name}.test"
 mkdir -p "${testdir}"
-mkdir -p "${ETCD_DIR}"
+mkdir -p "${ETCD_DATA_DIR}"
```

Comment:
unclear about this ... `$ETCD_DIR` is never set or used in Origin or in vagrant-openshift... I'm assuming because it i used as the data directory for etcd it should be `$ETCD_DATA_DIR`